cmd/cluster-capi-operator/main.go

```diff
@@ -155,6 +155,7 @@ func main() {
 		DefaultNamespaces: map[string]cache.Config{
 			*managedNamespace:                {},
 			secretsync.SecretSourceNamespace: {},
+			"kube-system":                    {}, // For fetching cloud credentials.
```

Comment:
This will never change?

Reply:
Correct, it is the location where vsphere credentials are stored in OCP

Reply:
Is there a reason not to use a shared constant for it here and in infracluster_controller.go?